lib/Member.js

```diff
@@ -1,11 +1,22 @@
-import UserInfo from './UserInfo';
+const {
```

Comment:
Fixed in: https://github.com/ccm-innovation/react-native-twilio-chat/pull/25/commits/9a62714ee9f8c2a48be22f03ab7ad912adf1012a